File: lib/models/art_data.dart
import 'dart:convert';

import 'package:camerapicoart/models/size.dart';

class ArtData {
  final String url;
  final String token;
  final String userId;
  final int artId;
  final Size size;

  ArtData({required this.url, required this.token, required this.userId, required this.artId, required this.size});

  String toJsonString() {
    return jsonEncode(toJson());
  }

  Map<String, dynamic> toJson() {
    return {
      'url': url,
      'token': token,
      'userId': userId,
      'artId': artId,
      'size': size.toJson(),
    };
  }
}

File: unity/AR Camera/Assets/Scripts/Models/ArtData.cs
using System;
using Newtonsoft.Json;

namespace Models
{
    [Serializable]
    public class ArtData
    {
        [JsonProperty("url")] public string Url { get; set; }
        [JsonProperty("token")] public string Token { get; set; }
        [JsonProperty("userId")] public string UserId { get; set; }
        [JsonProperty("artId")] public int ArtId { get; set; }
        [JsonProperty("size")] public Size Size { get; set; }
    }
}